src/utils/openmc/material.py

```diff
@@ -76,7 +76,9 @@ def set_id(self, material_id=None):
 
         # If the Material already has an ID, remove it from global list
         if not self._id is None:
-            MATERIAL_IDS.remove(self._id)
+            try:
```

Comment:
When testing before creating this PR I used a fresh clone of the default branch of https://github.com/mit-crpg/OpenCG.